template/contracts/src/BuyMeACoffee.sol

```diff
@@ -64,7 +65,7 @@ contract BuyMeACoffee {
      * @param  message The message of the user
      * (Note: Using calldata for gas efficiency)
      */
-    function buyCoffee(string calldata userName, string calldata twitterHandle, string calldata message) public payable {
+    function buyCoffee(int numCoffees, string calldata userName, string calldata twitterHandle, string calldata message) public payable {
         if (msg.value < price) {
```

Comment:
Buying more than one coffee at a time should reflect in the price the caller has to pay

Reply:
This should be a constraint on the contract and not just the frontend, since the UI is just an interface to the smart contract that anyone could call directly via an interface like Etherscan, Remix IDE, etc.

Reply:
@awilliams1-cb good idea. Would you like to put up a PR for the constraint?

Reply:
I can get that in! I think this will be required, shouldnt be that hard to do